install.sh

```diff
@@ -178,9 +178,18 @@ if [ ! -f "$RELAY_CREDENTIALS_JSON" ]; then
     echo ""
     echo "Generating Relay credentials..."
 
-    $dcr --user $(id -u) relay --config /etc/relay credentials generate --overwrite
-    chmod a+r $RELAY_CREDENTIALS_JSON
+    # We need the ugly hack below as `relay generate credentials` tries to read the config and the credentials
+    # even with the `--stdout` and `--overwrite` flags and then errors out when the credentials file exists but
+    # not valid JSON.
+    $dcr --no-deps --entrypoint /bin/bash relay -c "mkdir /work/.relay && cp /etc/relay/config.yml /work/.relay/config.yml && /bin/relay credentials generate > /dev/null && cat /work/.relay/config.yml" > "$RELAY_CREDENTIALS_JSON"
```

Comment:
>I wonder if you could just use /bin/relay credentials generate --stdout instead of generate && cat? There will be some additional output on stderr in either case, btw.

See my comment here please: I tried using `stdout` first but it fails if there's already a `credentials.json` file that is not valid. In our case I'd do `relay credentials generate > relay/credentials.json` which causes the `credentials.json` to be created as an empty file before `relay` runs and then `relay` complains about invalid config files as it cannot read an empty file as JSON.

Reply:
Gotcha, must be a misunderstanding on my end. I thought that it doesn't matter whether you run with or without `--stdout` as to whether the config is loaded beforehand. I'll follow up in Relay and make sure this works as expected.